src/index.js

```diff
@@ -381,4 +385,25 @@ export default class SauceLabs {
             throw new Error(`There was an error uploading assets: ${err.message}`)
         }
     }
+
+    async _createResultJob(parameters) {
+        const { servers, basePath, method, endpoint } = PROTOCOL_MAP.get('createResultJob')
+        const uri = getAPIHost(servers, basePath, this._options) + endpoint
+ 
+        try {
+            const res = await this._api(uri, {
+                method,
+                hooks: {
+                    beforeRequest: [
+                        options => {
+                            options.body = parameters;
+                        }
+                    ]
+                }
+           })
+           return res.body;
+        } catch (err) {
+           throw new Error(`There was an error create test results: ${err.message}`)
+        }
+    }
```

Comment:
This should not be needed. Based on the OpenAPI spec the library should be able to make this request as part of the proxy.

Reply:
Fixed! Now it works perfectly!